webpack.dev.config.js

```diff
@@ -93,7 +100,29 @@ export default merge(commonConfig, {
           },
         ],
       },
+      {
+        test: /\.css$/,
+        include: path.resolve(__dirname, 'node_modules/monaco-editor'),
+        use: ['style-loader', 'css-loader'],
+      },
     ],
   },
-  plugins: [new ReactRefreshWebpackPlugin(), new ForkTsCheckerWebpackPlugin({ devServer: false })],
+  plugins: [
+    new ReactRefreshWebpackPlugin(),
+    new ForkTsCheckerWebpackPlugin({ devServer: false }),
+    new MonacoWebpackPlugin({
```

Comment:
this webpack plugin config is also based on what the OpenShift Console team is using on their side

Reply:
I think this should be part of common webpack, this plugin would be required for prod also.